pages/level/[id]/[slugName].tsx

```diff
@@ -90,23 +85,22 @@ interface LevelSWRProps {
 
 export default function LevelSWR({ level }: LevelSWRProps) {
   const router = useRouter();
-  const { id } = router.query;
-
-  if (router.isFallback || !id) {
+  if (router.isFallback || !level) {
     return <SkeletonPage/>;
   }
-
+  const id = level._id;
+  const levelProps = {level:level};
   return (
     <SWRConfig value={{ fallback: { [getSWRKey(`/api/level-by-id/${id}`)]: level } }}>
-      <LevelPage/>
+      <LevelPage {...levelProps} />
```

Comment:
Need to remove this still and add `useLevelByUsernameAndSlug` in the `LevelPage`